doc/services/zbus/index.rst

```diff
@@ -212,13 +417,44 @@ The following code defines and initializes a regular channel and its dependencie
     }
     K_THREAD_DEFINE(subscriber_task_id, 512, subscriber_task, NULL, NULL, NULL, 3, 0, 0);
 
-It is possible to add static observers to a channel using the :c:macro:`ZBUS_CHAN_ADD_OBS`. We call that a post-definition static observer. The command enables us to indicate an initialization priority that affects the observers' initialization order. The priority param only affects the post-definition static observers. There is no possibility to overwrite the execution sequence of the static observers.
+    ZBUS_MSG_SUBSCRIBER_DEFINE(my_msg_subscriber);
+    static void msg_subscriber_task(void *sub)
+    {
+            const struct zbus_channel *chan;
+
+            struct acc_msg acc = {0};
+
+            while (!zbus_sub_wait_msg(&my_msg_subscriber, &chan, &acc, K_FOREVER)) {
+                    if (&acc_chan == chan) {
+                            LOG_INF("From msg subscriber -> Acc x=%d, y=%d, z=%d", acc.x, acc.y, acc.z);
+                    }
+            }
+    }
+    K_THREAD_DEFINE(msg_subscriber_task_id, 1024, msg_subscriber_task, NULL, NULL, NULL, 3, 0, 0);
+
+
+
+It is possible to add static observers to a channel using the :c:macro:`ZBUS_CHAN_ADD_OBS`. We call
+that a post-definition static observer. The command enables us to indicate an initialization
+priority that affects the observers' initialization order. The sequence priority param only affects
+the post-definition static observers. There is no possibility to overwrite the message delivery
+sequence of the static observers.
 
 .. note::
-    It is unnecessary to claim/lock a channel before accessing the message inside the listener since the event dispatcher calls listeners with the notifying channel already locked. Subscribers, however, must claim/lock that or use regular read operations to access the message after being notified.
+   It is unnecessary to claim/lock a channel before accessing the message inside the listener since
+   the event dispatcher calls listeners with the notifying channel already locked. Subscribers,
+   however, must claim/lock that or use regular read operations to access the message after being
+   notified.
 
 
-Channels can have a ``validator function`` that enables a channel to accept only valid messages. Publish attempts invalidated by hard channels will return immediately with an error code. This allows original creators of a channel to exert some authority over other developers/publishers who may want to piggy-back on their channels. The following code defines and initializes a :dfn:`hard channel` and its dependencies. Only valid messages can be published to a :dfn:`hard channel`. It is possible because a ``validator function`` was passed to the channel's definition. In this example, only messages with ``move`` equal to 0, -1, and 1 are valid. Publish function will discard all other values to ``move``.
+Channels can have a ``validator function`` that enables a channel to accept only valid messages.
```

Comment:
I am confused about the use of "code" formatting for `validator function`, is this on purpose? I think you probably meant to use single backquotes?